publication.go

```diff
@@ -138,3 +139,9 @@ func (p *Publication) Duplicate() *Publication {
 
 	return pub
 }
+
+func (p *Publication) Reply(response *Publication) {
+	if p.replyCh != nil {
+		p.replyCh <- response
+	}
+}
```

Comment:
@primalmotion do you have any ideas for this method? 

Does a caller need to know whether the publication is expecting a response or should they just call it and let the method silently exit (if `p.replyCh` is `nil`)?

Reply:
It should return an error if it doesn't expect an response. Hidden things always lead to weird bugs

Reply:
Done 👍